dashboard/constants.bal

```diff
@@ -50,24 +50,28 @@ const HEADER_LIBRARY_MODULES_DASHBOARD = "| Level | Name | Latest Version | Buil
 const HEADER_EXTENDED_MODULES_DASHBOARD = "| Name | Latest Version | Build | Security Check | Code Coverage | Bugs | Open PRs | Load Test Results | GraalVM Check |";
 const HEADER_HANDWRITTEN_CONNECTOR_DASHBOARD = "| Name | Latest Version | Build | Security Check | Code Coverage | Bugs | Open PRs | Load Test Results | GraalVM Check |";
 const HEADER_GENERATED_CONNECTOR_DASHBOARD = "| Name | Latest Version | Build | Security Check | Bugs | Open PRs | GraalVM Check |";
+const HEADER_DRIVER_CONNECTOR_DASHBOARD = "| Name | Latest Version | Build | Security Check |";
 const HEADER_TOOLS_DASHBOARD = "| Name | Latest Version | Build | Security Check | Code Coverage | Bugs | Open PRs |";
 
 const HEADER_SEPARATOR_LIBRARY_MODULES = "|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|";
 const HEADER_SEPARATOR_EXTENDED_MODULES = "|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|";
 const HEADER_SEPARATOR_HANDWRITTEN_CONNECTORS = "|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|";
 const HEADER_SEPARATOR_GENERATED_CONNECTORS = "|:---:|:---:|:---:|:---:|:---:|:---:|:---:|";
+const HEADER_SEPARATOR_DRIVER_CONNECTORS = "|:---:|:---:|:---:|:---:|";
 const HEADER_SEPARATOR_TOOLS = "|:---:|:---:|:---:|:---:|:---:|:---:|:---:|";
 
 const TITLE_LIBRARY_MODULES = "### Ballerina Modules";
 const TITLE_EXTENDED_MODULES = "### Ballerina Extended Modules";
 const TITLE_HANDWRITTEN_CONNECTORS = "### Ballerina Handwritten Connector Modules";
 const TITLE_GENERATED_CONNECTORS = "### Ballerina Generated Connector Modules";
+const TITLE_DRIVER_CONNECTORS = "### Ballerina Driver Modules";
```

Comment:
It seems we’ve used 2 naming conventions ("driver modules" and "driver connectors") throughout the changes and IMO it’s better to stick with one for consistency. I'm +1 for the term "driver modules". wdyt?

```suggestion
const TITLE_DRIVER_MODULES = "### Ballerina Driver Modules";
```

Reply:
+1. I renamed it to modules, must have missed in some places. I will update